### PR TITLE
[UX] Further Improvements

### PR DESCRIPTION
**Pull request by brxken128**
This implements the changes specified in #204.

A lot of print statements have been removed, and I feel as though I prefer it this way as it matches other command-line applications.

Most command-line applications don't print to the console unless they've errored, and I feel that's a good thing to stick by.

Obviously this is very subjective, so feel free to object :)

### CODE REVIEW

File: dexios/src/cli.rs [hunk 1]
```diff
@@ -69,12 +69,10 @@ pub fn get_matches() -> clap::ArgMatches {
                 .help("Force all actions"),
         )
         .arg(
-            Arg::new("aead")
```
Comment (pleshevskiy) [inline dexios/src/cli.rs:72]: You should add 'deoxys' argument if `deoxis-ii-256` feature is enabled. Or remove deoxys feature https://github.com/brxken128/dexios/blob/master/dexios/Cargo.toml#L21

Comment (pleshevskiy) [reply dexios/src/cli.rs:72]: > The deoxys feature isn't enabled for the CLI app though, is it? > I'm not too sure I see the point in making a switch for a feature that isn't enabled.

You can enable deoxys if you run cli with `--features deoxys-ii-256`

Comment (pleshevskiy) [reply dexios/src/cli.rs:72]: > Until performance of Deoxys improves, I'm not comfortable adding the option to encrypt with it back in though.

ok 👌🏼

**Comment by brxken128** [comment]
@pleshevskiy I removed the Deoxys feature. The feature was probably a bad idea as it needed to be enabled in order to decrypt files that were encrypted with Deoxys.

I'm still not going to add the support to *encrypt* with it (although I will do some testing to see how performance is doing), but we still need to provide the option to decrypt with it.

I also removed the visual feature by default, as this should clean CLI output up a lot as well. My goal is to make Dexios more UNIX-like, and only alert the user to something where it's required.

**Comment by pleshevskiy** [comment]
LGTM!

**Pull request merged**